.github/workflows/docker_push.yaml

```diff
@@ -0,0 +1,63 @@
+name: Create and push cuda118 + cuda120 docker images to this repo's packages
+
+on:
+  workflow_dispatch: {}
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-cuda-images:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - 
+        name: Checkout repository
+        uses: actions/checkout@v4
+  
+      - 
+        name: Log in to the Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      #CUDA120 steps
+      - 
+        name: Extract cuda120 image metadata
+        id: meta_120
+        uses: docker/metadata-action@v3
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+          tags: cuda120
```

Comment:
Is it possible to use multiple tags here? We would like to have a default `latest` tag that defaults to cuda 12.0 package. This is a common practice with other docker images.